custom/enikshay/ucr/reports/tb_notification_register.json

```diff
@@ -41,9 +41,10 @@
             {
                 "compare_as_string": false,
                 "datatype": "string",
-                "type": "dynamic_choice_list",
+                "type": "multi_field_dynamic_choice_list",
                 "slug": "village",
                 "field": "person_owner_id",
```

Comment:
@NoahCarnahan I can't make field property not required due to this line https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/reports/factory.py#L22. Is there any reason to not use report filter factory in that line?